src/scripts/main.js

```diff
@@ -1 +1,31 @@
 'use strict';
+
+const menu = document.querySelector('.btn');
+const mobileNavigation = document.querySelector('.nav');
+
+menu.addEventListener('click', (event) => {
+  mobileNavigation.classList.toggle('nav--active');
+  event.stopImmediatePropagation();
+});
+
+const form = document.querySelector('.feedback__form');
+const inputs = document.querySelectorAll('.feedback__form-input');
+
+form.addEventListener('submit', (e) => {
```

Comment:
Не скорочуй назви змінних
```suggestion
form.addEventListener('submit', (event) => {
```